src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -37,9 +38,11 @@
 
 TOOLS_AVAILABLE = ['gtdb_tk']  # TODO: fix checkm2 container bug
 
-# All nodes in Perlmutter have identical computational resources
-CHUNK_SIZE = {'checkm2': 5000,
-              'gtdb_tk': 1000}
+# estimated execution time (in minutes) for each tool to process a chunk of data
+TASK_META = {'checkm2': {'chunk_size': 5000, 'exe_time': 60},
+             'gtdb_tk': {'chunk_size': 1000, 'exe_time': 90}}
```

Comment:
I assume you'll check these? They probably need a bit of a fudge factor to be sure jobs don't fail

Reply:
yea. I will test it for sure. According to my previous benchmarks, GTDB-TK takes about 30 mins for 1000 genomes with 128 cpus.

Reply:
:+1: